systemtests/README.rst

```diff
@@ -22,15 +22,18 @@ Set up the tests this way::
 
 That creates files in directories under ``data/``.
 
-You only need to set up the tests once to run the system tests against all environments.
+You only need to set up the tests once to run the system tests against all
+environments.
```

Comment:
While we're here, I realized it's not mentioned that the local Tecken stack needs to be running for the local tests to work (and that means `./slick.sh` as well as `make run`). I only did the former, and hit an error since the `web` container wasn't running. Is it worth mentioning this in the README?

Reply:
I'll add a note that you have to follow the local dev environment set up process.